css/types/attr.json

```diff
@@ -79,9 +79,9 @@
             }
           }
         },
-        "type-or-unit": {
+        "type_function": {
           "__compat": {
-            "description": "&lt;type-or-unit&gt;",
+            "description": "`type(<syntax>)` function",
```

Comment:
This looks good. 
1. I assume we don't need to track the historical syntax?
2. Normally  BCD updates automatically to link to associated reference
   - so in this case the closes thing we have is https://developer.mozilla.org/en-US/docs/Web/CSS/attr#attr-type
   - But the docs should be updated so that `type()` is a definition list so it is linkable.
   - The question is, how will BCD know to link to the reference when it is created in docs? Should we add a link to MDN docs manually?